wp-content/themes/borderzine/inc/widgets/class-borderzine-3-col-widget.php

```diff
@@ -0,0 +1,388 @@
+<?php
+/**
+ * Borderzine 3-Column recent posts widget and associated functions
+ *
+ * Copied from Largo Recent Posts
+ */
+
+/**
+ * Borderzine 3-Column recent posts widget
```

Comment:
@benlk Also a heads up, just noticed that this comment is duplicated.

Reply:
Resolved in https://github.com/INN/umbrella-borderzine/pull/67/commits/f03b6b3c7797ce78a627772d1ca111f48230c914